Protocols/protocol_to_config.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
This file generates `#define` to be writtent o `protocol.h` by `cmake`. This is developer only change and user may not worry about it.